### PR TITLE
feat: add info about convert tasks into issues

### LINKED ISSUE #24862: Explain what conditions must be met to show hover icon to create issue behind tasklist items
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists

### What part(s) of the article would you like to see updated?

The docs say:

> If a task requires further tracking or discussion, you can convert the task to an issue by hovering over the task and clicking _button-icon-here_ in the upper-right corner of the task.

The docs don't explain if there are any rules you need to follow to allow GitHub to show that hover icon, like:

- only put issue or PR link after the tasklist
- don't use special characters like `(` or `)`
- don't do ...

Or maybe this is a "rights" issue where you need a certain level of access to the repository before you see that button?

### Additional information

Why I'm asking about any possible rules: @viceice (maintainer of the repository) is **not** seeing the hover icon behind items in their tasklist:

- https://github.com/containerbase/base/issues/875

With the current docs I can't determine if this is:

- bad formatting of the tasklist
- a problem with access rights to the repository
- bug in GitHub

----
_Update from a maintainer_

## Content change suggested to fix this issue

We need to make it clearer to users how to convert a tasklist item to an issue. We'll do this by adding a brief new section to the article [About task lists](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists).

1. After the section "Reordering tasks", add a new heading level 2 "Converting tasks into issues".
2. Add a brief explanation of how to convert a task into an issue.
3. Add the following screenshot to the folder for images about writing: https://github.com/github/docs/tree/main/assets/images/help/writing.
    task-list-convert
4. Include the image below the explanation and make sure you add suitable alt text (see [Images](https://docs.github.com/en/contributing/writing-for-github-docs/style-guide#images)). 
 
You may find it useful to look at the alt text for other images on the page for inspiration, for example: 
    https://github.com/github/docs/blob/275f369a2b5e01bfd108f8f957383d61ba5746f2/content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md?plain=1#L64

### PR DESCRIPTION
**Pull request by CBID2**
### Why:
This PR adds a section on how to convert tasks created on issues into their own issues. This helps maintainers learn another way to organize their issues.

### Closes: 
Closes #24862 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @cmwilson21! :) As promised, I made the PR.

**Comment by CBID2** [comment]
Ummm @mchammer01 & @cmwilson21, all of the tests failed.

**Review by mchammer01 — COMMENTED** [review]
Let's try and fix the check failures.

File: content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md
Comment (mchammer01) [inline content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md:74]: Remove superfluous newline that causes a failure of the content-linter check.
```suggestion
```

**Comment by mchammer01** [comment]
I'll unassign myself for now, until we decide how to proceed with this PR.
@CBID2 - I do believe I fixed the check failures 😃

**Comment by CBID2** [comment]
> I'll unassign myself for now, until we decide how to proceed with this PR.
> 
> @CBID2 - I do believe I fixed the check failures 😃 

Yup you did. If you're a maintainer, I think you can approve of it.

**Comment by mchammer01** [comment]
@CBID2 👋🏻 - thanks for your note. I haven't reviewed your PR yet because I think, although it looks good, and contains a useful new procedural section, it doesn't answer the part of the issue that mention explaining what conditions must be met to show hover icon to create issue behind tasklist items. I have ask folks who are more familiar with task lists than me if they can help.
Thanks for your patience with us ❤️

File: content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md
Comment (stevecat) [inline content/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists.md:72]: Just making a few small tweaks here, updating the screenshot alt text, and using an octicon so we don't need to visually describe the icon. I'll apply this suggestion myself 😸 

```suggestion
You can also convert tasks into issues. First, hover over one of the items in your task list and then click {% octicon "issue-opened" aria-label="Convert to issue" %}.

 {% ifversion fpt or ghec %} ![Screenshot of a {% data variables.product.prodname_dotcom %} issue showing two tasks. The"Convert to issue" icon is highlighted with an orange outline.](/assets/images/help/writing/convert-task-lists-into-issues.png){% endif %}
```

**Review by guntrip — APPROVED** [review]
This looks great, @CBID2, thanks so much for picking this up! ✨ 

It looks like Felicity tested for reasons the icon might not appear in the associated issue and didn't find any. 

I made a few little changes via a suggested comment, and included the octicon directly rather than describing it which I think will make this procedural more accessible; the octicon includes the same alt text as the UI, so a screenreader will describe it in the same way.

We've got two unrelated test failures, for our API docs. I'll see if this will merge, if not, I'll ask our engineers to take a look.

Thank you again! ✨

**Pull request merged**
